src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -266,7 +266,8 @@ public OffenderDetailSummary getOffenderDetails(
         final @RequestParam(value = "attended", required = false) Optional<Boolean> attended,
         final @RequestParam(value = "complied", required = false) Optional<Boolean> complied,
         final @RequestParam(value = "nationalStandard", required = false) Optional<Boolean> nationalStandard,
-        final @RequestParam(value = "outcome", required = false) Optional<Boolean> outcome) {
+        final @RequestParam(value = "outcome", required = false) Optional<Boolean> outcome,
+        final @ApiParam(name = "include", value = "Contacts to include. Can be a contact type code, prefixed with 'type_' or appointments", example = "type_ccmp", allowMultiple = true) @RequestParam(value = "include", required = false) Optional<List<String>> include) {
```

Comment:
Could this just be a List and treat an empty list as not present ?

Reply:
@andymarke looks like the existing contactTypes was used as a template for this new param so although an empty list would work fine, it would be inconsistent.